go/example_code/s3/delete-buckets-by-prefix/s3_delete_buckets_test.go

```diff
@@ -0,0 +1,146 @@
+/*
+   Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This file is licensed under the Apache License, Version 2.0 (the "License").
+   You may not use this file except in compliance with the License. A copy of
+   the License is located at
+
+    http://aws.amazon.com/apache2.0/
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations under the License.
+*/
+package main
+
+import (
+    "fmt"
+    "os"
+    "strconv"
+    "strings"
+    "testing"
+    "time"
+
+    guuid "github.com/google/uuid"
```

Comment:
Any particular reason we bring this in as guuid instead of the canonical uuid?

Reply:
Nope, I'll fix it.